MediaBrowser.Controller/Extensions/ConfigurationExtensions.cs

```diff
@@ -22,6 +28,15 @@ public static class ConfigurationExtensions
         /// </summary>
         public const string PlaylistsAllowDuplicatesKey = "playlists:allowDuplicates";
 
+        /// <summary>
+        /// Gets a value indicating whether the application should not host static web content from the <see cref="IConfiguration"/>.
```

Comment:
```suggestion
        /// Gets a value indicating whether the application should host static web content from the <see cref="IConfiguration"/>.
```
This parameter is in positive form :) but you copied from a negated value.

Reply:
Fixed